agent/apps/ai_guardrails/app_code/models.py

```
@@ -0,0 +1,168 @@
+    )
+    priority = models.IntegerField(
+        default=100,
+        validators=[MinValueValidator(0)],
+        verbose_name="اولویت",
+    )
+    conditions = models.JSONField(default=dict, blank=True, verbose_name="شرایط")
+
+    class Meta:
+        verbose_name = "سیاست گاردریل"
+        verbose_name_plural = "سیاست‌های گاردریل"
+        ordering = ["priority", "-created_at"]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)